### PR TITLE
[FEATURE] Add integration test for zep sqlalchemy datasource with renderering.

### PR DESCRIPTION
**Pull request by billdirks**
Add test for rendered content for new experimental postgres datasource.

- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added [unit tests](https://docs.greatexpectations.io/docs/contributing/contributing_test#writing-unit-and-integration-tests) where applicable and made sure that new and existing tests are passing.
- [x] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

File: tests/experimental/datasources/test_integration_sql_datasource.py [hunk 1]
```diff
@@ -21,6 +22,9 @@ def test_run_checkpoint_and_data_doc(empty_data_context):
     )
     context: DataContext = empty_data_context
 
+    if include_rendered_content:
+        context.variables.include_rendered_content.globally = True
```
Comment (NathanFarmer) [inline tests/experimental/datasources/test_integration_sql_datasource.py:30]: Why not just `context.variables.include_rendered_content.globally = include_rendered_content`?

File: tests/experimental/datasources/test_integration_sql_datasource.py [hunk 2]
```diff
@@ -88,19 +92,58 @@ def test_run_checkpoint_and_data_doc(empty_data_context):
...
         )
 
+        if include_rendered_content:
+            # There is a prescriptive atomic renderer on r.expectation_config
+            num_prescriptive_renderer = len(r.expectation_config.rendered_content)
+            assert (
+                num_prescriptive_renderer == 1
+            ), f"Expected exactly 1 rendered content, found {num_prescriptive_renderer}"
+            rendered_content = r.expectation_config.rendered_content[0]
+            assert rendered_content.name == "atomic.prescriptive.summary"
```
Comment (NathanFarmer) [inline tests/experimental/datasources/test_integration_sql_datasource.py]: You can use `AtomicPrescriptiveRendererType.SUMMARY` from `great_expectations/render/components.py` here.

File: tests/experimental/datasources/test_integration_sql_datasource.py [hunk 3]
```diff
@@ -88,19 +92,58 @@ def test_run_checkpoint_and_data_doc(empty_data_context):
...
+                ]
+            )
+
+            # There is a diagnostic atomic renderer on r, a validation result result.
+            num_diagnostic_render = len(r.rendered_content)
+            assert (
+                num_diagnostic_render == 1
+            ), f"Expected 1 diagnostic renderer, found {num_diagnostic_render}"
+            diagnostic_renderer = r.rendered_content[0]
+            assert diagnostic_renderer.name == "atomic.diagnostic.observed_value"
```
Comment (NathanFarmer) [inline tests/experimental/datasources/test_integration_sql_datasource.py]: You can use `AtomicDiagnosticRendererType.OBSERVED_VALUE` here.

**Review by NathanFarmer — APPROVED** [review]
Approved with a few non-blocking suggestions.

File: tests/experimental/datasources/test_integration_sql_datasource.py (hunk 1 above)
Comment (billdirks) [reply tests/experimental/datasources/test_integration_sql_datasource.py:30]: I wanted to have a basic test where I didn't set anything "extra". That is I wanted to be able to verify I could run a checkpoint and create datadocs without setting this. Maybe it doesn't matter though.

File: tests/experimental/datasources/test_integration_sql_datasource.py (hunk 2 above)
Comment (billdirks) [reply tests/experimental/datasources/test_integration_sql_datasource.py]: Done

File: tests/experimental/datasources/test_integration_sql_datasource.py (hunk 3 above)
Comment (billdirks) [reply tests/experimental/datasources/test_integration_sql_datasource.py]: Done.

**Review by billdirks — COMMENTED** [review]
Thanks for the review!

**Pull request merged**